app/validators/children_validator.rb

```diff
@@ -0,0 +1,12 @@
+# See README.md for copyright details
+class ChildrenValidator < ActiveModel::EachValidator
```

Comment:
I think we should change the name of this class.
Is it really validates some value or is it just checking some other validation's result?
Or can we move this code where the validation is happening?